src/components/DrawerSection.tsx

```diff
@@ -70,18 +71,26 @@ const DrawerSectionEntryWrapper = (props: StackProps) => (
 
 export const DrawerSectionEntry = ({
     title,
+    titleStyle,
     children,
     alignItems,
+    icon,
 }: {
     title: string;
+    titleStyle?: 'code';
     children?: React.ReactNode;
     alignItems?: 'flex-start' | 'center' | 'baseline';
+    icon?: JSX.Element;
 }): JSX.Element => {
     return (
         <DrawerSectionEntryWrapper alignItems={alignItems || 'center'}>
-            <Text whiteSpace="nowrap" width="120px" flexShrink="0">
-                {capitalize(title)}
-            </Text>
+            {icon}
+            {titleStyle === 'code' && <Code fontSize="xs">{title}</Code>}
```

Comment:
How about using this style also for the output of the parent_task (in TaskInputRepresentation, where we show 'parent_task_output_identifier') ?